ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -156,3 +156,4 @@ sunbird_portal_session_secret={{sunbird_portal_session_secret}}
 
 #release-4.0.0
 sunbird_base_proto={{sunbird_base_proto | default(proto)}}
+ML_SERVICE_BASE_URL={{ML_SERVICE_BASE_URL}}
```

Comment:
Keep the right hand side variable as small letters. And what's the use of the variable? I meant the variable name is not contextful.

Reply:
@rjshrjndrn Right side variable change done. This URL is specific to ML backend service and this variable will be removed by 4.2 release.

Reply:
Which backend service specifically? 
Means ml has multiple services running, like project, reports, etc.
So if it's pointing to a specific service then it'll be better to have variables like `{{ml_project_service_url | default('project-service:3000'}}`

The default value because, project service is running in Kubernetes, and the private endpoint for that service not gonna change.